dspace/config/local.cfg.EXAMPLE

```diff
@@ -247,3 +247,30 @@ db.schema = public
 
 # Maximum size of a multipart request (i.e. max total size of all files in one request)
 #spring.servlet.multipart.max-request-size = 512MB
+
+#---------------------------------------------------------------#
+#-----------------STORAGE CONFIGURATIONS------------------------#
+#---------------------------------------------------------------#
```

Comment:
I would recommend removing all of these from `local.cfg.EXAMPLE`.  We are trying to keep this example file minimal...notice it doesn't list every configuration.

Instead, if you want to mention S3 in this file, I'd recommend adding a comment above the `assetstore.dir` setting in `local.cfg.EXAMPLE` to say something like:
```
# Additional storage options (e.g. Amazon S3) are available in `assetstore.cfg`
```